app/views/scripts/production_deploy.erb

```diff
@@ -46,6 +46,7 @@ EOF
   for herokuapp in ${HEROKU_APPS}
   do
       heroku config:add HEROKU_APP_NAME="${herokuapp}" --app ${herokuapp}
+      heroku config:set BUNDLE_WITHOUT="development:test darwin"
```

Comment:
```
heroku config:set BUNDLE_WITHOUT="development:test:darwin"
```

might be correct instead of space? https://devcenter.heroku.com/articles/bundler-configuration